src/test/java/ticTacToe/guiAppTest/ActionSetterTest.java

```diff
@@ -0,0 +1,20 @@
+package ticTacToe.guiAppTest;
+
+import org.junit.Test;
+import ticTacToe.grid.Grid;
+import ticTacToe.guiApp.ActionSetter;
+
+import static org.junit.Assert.assertTrue;
+
+public class ActionSetterTest {
+
+    @Test
+    public void setsActionOnButton() {
+        ActionSetter actionSetter = new ActionSetter(new Grid(3));
+        ButtonDouble button = new ButtonDouble();
+
+        actionSetter.addClickHandler(button, "1");
+
+        assertTrue(button.wasCalled);
+    }
+}
```

Comment:
You could write a test for a button which already has X or O as text. And assert that the text on it doesn't change.